$OpenBSD: patch-servers_visual_shader_language_cpp,v 1.1 2019/08/16 15:38:15 thfr Exp $

Fix error with ports-gcc:
  error: converting to 'bool' from 'std::nullptr_t' requires direct-initialization [-fpermissive]

Index: servers/visual/shader_language.cpp
--- servers/visual/shader_language.cpp.orig
+++ servers/visual/shader_language.cpp
@@ -1869,11 +1869,11 @@ const ShaderLanguage::BuiltinFuncDef ShaderLanguage::b
 
 bool ShaderLanguage::_validate_function_call(BlockNode *p_block, OperatorNode *p_func, DataType *r_ret_type) {
 
-	ERR_FAIL_COND_V(p_func->op != OP_CALL && p_func->op != OP_CONSTRUCT, NULL);
+	ERR_FAIL_COND_V(p_func->op != OP_CALL && p_func->op != OP_CONSTRUCT, false);
 
 	Vector<DataType> args;
 
-	ERR_FAIL_COND_V(p_func->arguments[0]->type != Node::TYPE_VARIABLE, NULL);
+	ERR_FAIL_COND_V(p_func->arguments[0]->type != Node::TYPE_VARIABLE, false);
 
 	StringName name = static_cast<VariableNode *>(p_func->arguments[0])->name.operator String();
 
